frontend/src/components/personal/ProfilePreferencesPage.styles.tsx

```diff
@@ -38,5 +39,37 @@ export const useStyles = makeStyles((theme) => ({
     },
     textfields: {
       margin: 15,
-    }
-  }))
\ No newline at end of file
+    },
+    
+  }))
+
+  export const useStyleModal = makeStyles((theme: Theme) =>
+  createStyles({
+    paper: {
+      position: 'absolute',
+      width: 400,
+      backgroundColor: theme.palette.background.paper,
+      border: '1px dotted #000',
+      boxShadow: theme.shadows[5],
+      padding: theme.spacing(2, 4, 3),
+      fontFamily: 'Roboto',
+      lineHeight: '1.5rem'
+    },
+  }),
+)
+
+export function rand() {
```

Comment:
Why does this function in *styles* ?

Reply:
moved this function inside getModalStyle